electrumsv/wallet.py

```diff
@@ -3713,18 +3710,19 @@ def find_account_for_extended_public_key(self, extended_public_key: XPublicKey)
                     return account
         return None
 
-    # TODO(1.4.0) Remove when we have replaced with a reference server equivalent.
-    # def undo_verifications(self, above_height: int) -> None:
-    #     '''Called by network when a reorg has happened'''
-    #     if self._stopped:
-    #         self._logger.debug("undo_verifications on stopped wallet: %d", above_height)
-    #         return
+    def on_reorg(self, orphaned_block_hashes: List[bytes]) -> None:
```

Comment:
Make this asynchronous, make `set_transactions_reorged` a method on the async db functions class and call it async. I do not know why this is running in a thread.